core/src/main/scala/org/apache/spark/ui/jobs/AllStagesPage.scala

```diff
@@ -37,12 +37,18 @@ private[ui] class AllStagesPage(parent: StagesTab) extends WebUIPage("") {
       val numCompletedStages = listener.numCompletedStages
       val failedStages = listener.failedStages.reverse.toSeq
       val numFailedStages = listener.numFailedStages
+      val pendingStages = listener.pendingStages.values.toSeq
```

Comment:
can you move this up with `activeStages` to make the declarations grouped properly?